projects/vdk-plugins/vdk-notebook/tests/jobs/ingest-data-flow-job/steps.ipynb

```diff
@@ -0,0 +1,63 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "654db028-2d0e-4f95-a377-e1fac1255f08",
+   "metadata": {
+    "tags": [
+     "vdk"
+    ],
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
```

Comment:
Here I see a specific format for the input, please provide a good documentation for it both in the code and in the user docs